### PR TITLE
[Super-Ready for review]Reworks healing symptoms into conditional healing symptoms

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
balance: Viruses' healing symptoms have been reworked!
del: All existing healing symptoms have been removed in favour of new ones.
del: Weight Even and Weight Gain have been removed, so hunger can be used for balancing in symptoms.
add: Starlight Condensation heals toxin damage if you're in space using starlight as a catalyst. Being up to two tiles away also works, as long as you can still see it, but slower.
add: Toxolysis (level 7) rapidly cleanses all chemicals from the body, with no exception.
add: Cellular Molding heals brute damage depending on your body temperature: the higher the temperature, the faster the healing. Requires above-average temperature to activate, and the speed heavily increases while on fire.
add: Regenerative Coma (level 8) causes the virus to send you into a deep coma when you are heavily damaged (>70 brute+burn damage). While you are unconscious, either from the virus or from other sources, the virus will heal both brute and burn damage fairly quickly. Sleeping also works, but at reduced speed.
add: Tissue Hydration heals burn damage if you are wet (negative fire stacks) or if you have water in your bloodstream.
add: Plasma Fixation (level 8) stabilizes temperature and heals burns while plasma is in your body or while standing in a plasma cloud. Does not protect from the poisoning effects of plasma.
add: Radioactive Resonance gives a mild constant brute and burn healing while irradiated. The healing becomes more intense if you reach higher levels of radiation, but is still less than the alternatives.
add: Metabolic Boost (level 7) doubles the rate at which you process chemicals, good and bad, but also increases hunger tenfold.
/:cl:

Further suggestions welcome etc.
I hope it's enough to prevent virology from getting the axe.

### CODE REVIEW

**Comment by Spaceinaba** [comment]
this is sexy

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -305,25 +363,46 @@ Bonus
...
+	symptom_delay_max = 1
+	threshold_desc = "<b>Stage Speed 6:</b> Increases healing."
+
+/datum/symptom/heal/radiation/CanHeal(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	switch(M.radiation)
+		if(0)
+			return FALSE
+		if(1 to 350)
+			return 0.25
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: It'd probably be best if you can't heal unless you're getting some negative from the radiation.
`if(0 to RAD_MOB_SAFE) return FALSE`

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 2]
```diff
@@ -305,25 +363,46 @@ Bonus
...
+			return 0.5
+		if(751 to 1250)
+			return 0.75
+		if(1251 to 2000)
+			return 1
+		else
+			return 1.5
+
+/datum/symptom/heal/radiation/Heal(mob/living/carbon/M, datum/disease/advance/A, actual_power)
+	var/heal_amt = 1 * actual_power
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: `heal_amt = actual_power`

**Comment by ChangelingRain** [comment]
Weight gain also has to go if you want to use nutrition to balance symptoms.

**Comment by XDTM** [comment]
sure

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 3]
```diff
@@ -256,18 +299,33 @@ Bonus
...
-/datum/symptom/heal/burn/plus/Heal(mob/living/carbon/M, datum/disease/advance/A)
-	var/heal_amt = 4 * power
+/datum/symptom/heal/plasma/CanHeal(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	var/datum/gas_mixture/environment
+	var/list/gases
+
+	. = 0
+
+	if(M.loc)
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: store get_turf(M) then check if it's an open turf before getting the air

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 4]
```diff
@@ -380,9 +380,9 @@ Bonus
 	switch(M.radiation)
 		if(0)
 			return FALSE
-		if(1 to 350)
+		if(1 to RAD_MOB_SAFE)
 			return 0.25
-		if(351 to 750)
+		if(RAD_MOB_SAFE + 1 to 750)
```
Comment (ChangelingRain) [inline code/datums/diseases/advance/symptoms/heal.dm]: you don't need the +1

Comment (ChangelingRain) [reply code/datums/diseases/advance/symptoms/heal.dm]: or to be accurate, with it decimal radiation values between RAD_MOB_SAFE and RAD_MOB_SAFE + 1 will trigger the else and give 1.5

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 5]
```diff
@@ -49,57 +53,63 @@ Toxin Filter
...
+	else
+		for(var/turf/T in view(M, 2))
+			if(istype(T, /turf/open/space))
+				return power * 0.3
+
+/datum/symptom/heal/toxin/Heal(mob/living/M, datum/disease/advance/A, actual_power)
+	var/heal_amt = 1 * actual_power
 	if(M.toxloss > 0 && prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#66FF99")
+		new /obj/effect/temp_visual/heal(M.loc, "#66FF99")
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: get_turf(M)

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 6]
```diff
@@ -142,57 +168,62 @@ Bonus
 			M.update_damage_overlays()
 
 	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#FF3333")
+		new /obj/effect/temp_visual/heal(M.loc, "#FF3333")
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: get_turf(M)

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 7]
```diff
@@ -142,57 +168,62 @@ Bonus
...
 		return
 
 	for(var/obj/item/bodypart/L in parts)
-		if(L.heal_damage(heal_amt/parts.len, 0))
+		if(L.heal_damage(heal_amt/parts.len, heal_amt/parts.len))
 			M.update_damage_overlays()
 
 	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#CC1100")
+		new /obj/effect/temp_visual/heal(M.loc, "#CC1100")
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: get_turf(M)

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 8]
```diff
@@ -236,7 +279,7 @@ Bonus
 			M.update_damage_overlays()
 
 	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#FF9933")
+		new /obj/effect/temp_visual/heal(M.loc, "#FF9933")
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: get_turf(M)

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 9]
```diff
@@ -284,7 +342,7 @@ Bonus
 			M.update_damage_overlays()
 
 	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#CC6600")
+		new /obj/effect/temp_visual/heal(M.loc, "#CC6600")
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: get_turf(M)

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 10]
```diff
@@ -305,25 +363,46 @@ Bonus
...
+
+	if(!parts.len)
+		return
+
+	for(var/obj/item/bodypart/L in parts)
+		if(L.heal_damage(heal_amt/parts.len, heal_amt/parts.len))
+			M.update_damage_overlays()
+
+	if(prob(base_message_chance) && !hide_healing)
+		new /obj/effect/temp_visual/heal(M.loc, "#66ff22")
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: get_turf(M)

**Review by ninjanomnom — DISMISSED** [review]

**Comment by XDTM** [comment]
the loc is intentional so you don't spit overlays outside of lockers while you're hiding

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 11]
```diff
@@ -142,57 +168,62 @@ Bonus
...
+	var/mob/living/M = A.affected_mob
+	if(M.IsUnconscious() || M.stat == UNCONSCIOUS)
+		return power
+	else if(M.stat == SOFT_CRIT)
+		return power * 0.5
+	else if(M.IsSleeping())
+		return power * 0.25
+	else if(M.getBruteLoss() + M.getFireLoss() >= 70)
+		to_chat(M, "<span class='warning'>You feel yourself slip into a regenerative coma...</span>")
+		M.Unconscious(450)
```
Comment (ninjanomnom) [inline code/datums/diseases/advance/symptoms/heal.dm]: This should probably be delayed and have a message so people have time to get somewhere safe or finish their current action.

**Comment by ninjanomnom** [comment]
You should be checking if loc is a turf then rather than creating effects in lockers

**Comment by XDTM** [comment]
they self-delete in a second or two anyway, it makes almost no difference

**Comment by ninjanomnom** [comment]
It's still bad practice to do needless operations like that, that's how you get bugs from people modifying your code in the future.

**Comment by XDTM** [comment]
Technically i'd want the animation to pop out as soon as you open the locker, only thing is that it's so short that it isn't particularly necessary. But in my opinion, if it was, it _should_ leave an effect to display on locker opening.

**Comment by ChangelingRain** [comment]
also, these animations shouldn't cause bugs if in weird places. if something handles them badly, that thing shouldn't.

**Comment by ExcessiveUseOfCobblestone** [comment]
Nice

**Comment by ExcessiveUseOfCobblestone** [comment]
Although at some point we need to move away from viro being a job that adds yet another avenue to heal your wounds when it's already piss easy to do

**Comment by XDTM** [comment]
I'll think of some other symptoms eventually, but this at least fixes the fully free healing issue.

**Comment by Ratvar** [comment]
Only to find out if you can lock people in regenerative coma stunlock with bad symptoms. And how it gonna balance with people effectively getting knocked into deepcrit with 70 damage if someone stealthly spreads regencoma.

**Comment by Doppeltongue** [comment]
These are at least very interesting. Maybe make healing symptom stats a little better, but I'm assuming those are all WIP. 

Possible suggestions:
Make holy water stronger than it is right now, because it does make you drunk, and being drunk is a pretty decent downside. 
Give the coma ability a play dead and deathgasp effect when you're in crit? You're still down, and people often try to kill corpses well in excess of 200 damage.
Radiation might need to also remove rads, and I'm not sure if there's any good source of self-inflicted radiation for it, so the healing might need to be stronger.
How about something like Metabolic Boost: increases the healing that nutriment gives you, but increases the rate that you get hungry?

**Comment by kevinz000** [comment]
coma + necro = ggnore?

**Comment by ChangelingRain** [comment]
regen coma is flat out unusable, since going into crit at 70 is hilariously lethal in any combat, even if it comes with healing.

**Comment by Ratvar** [comment]
really good as incapicating agent in now-nonlethal viruses, tho, so you can collect bodies and heal/convert.

**Comment by XDTM** [comment]
Changed coma a bit:
- The healing is much faster.
- The host has FAKEDEATH while it's active
- The coma ends when you're healthy

So now you're trading 30 health before crit (and soft crit) in exchange for healing while unconscious, which may not suit combat situations but it's much better for natural hazards. Also be prepared to be lynched as a ling.

**Comment by Ratvar** [comment]
Forever incapicating virus is even more easy to make now?

**Comment by Nabski89** [comment]
Are you planning any threshold things for these?

Toxin starlight is the easiest condition to meet, but also the least likely influence combat.
I like how you can't be healing brute and burn at the same time (Unless you're chugging water bottles).

Two of the level 8's are serious gamechangers where you would have to decide if you want it or not. Plasma Fixation slightly less so.

I feel like you removed the weight things without using them for any balance related things, but they're also shit virus's that fell into the "free thing that you would never not want" category.

**Comment by XDTM** [comment]
@Ratvar, now the effects ends when on full health, so you have to keep hitting them to keep them down. And if you had that possibility you could just, y'know, knock hem into crit.

**Comment by The-clown-known-as-Pepe** [comment]
Can we get a sympthom that processes all chems faster at cost of increased nutrition loss?

**Comment by XDTM** [comment]
Processes in the sense of removal or increasing the effectiveness?

**Comment by The-clown-known-as-Pepe** [comment]
In sense of like halving chem ticks so they act faster
It would be a nice double edged sword, being healed faster but dying faster from poison.

**Comment by XDTM** [comment]
I liked the idea and added it

**Comment by Nabski89** [comment]
Can't I just stick Cellular Molding on a virus with fever to activate it?

**Comment by XDTM** [comment]
You can! It's one slot less for other symptoms, though, and it won't heal much if you don't go over the damage threshold, which would then do burn damage instead

**Comment by Armhulen** [comment]
"i'm low on health, better find somewhere safe to activate my fleshme- coma"

this pr is rats

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 12]
```diff
@@ -21,8 +20,6 @@
 		hide_healing = TRUE
```
Comment (ChangelingRain) [inline code/datums/diseases/advance/symptoms/heal.dm]: you should probably remove this since, well, these are all much more limited symptoms

**Comment by ChangelingRain** [comment]
regen coma is still pretty bad, since having 70 health before crit makes you way more fragile.
stuns are death, and a stun that only needs you to get hurt more than it heals per tick(4 or 8 brute and burn each tick is a lot, but not hard to keep down)

**Comment by XDTM** [comment]
@Armhulen seems to think that it's really op because you can take a regenerating coma anytime
@ChangelingRain seems to think it sucks because of the effective reduced health

I think i've reached a symptom that is good but balanced by its downsides

**Comment by Jalleo** [comment]
Yeah its situatioally good and bad. i approve

**Comment by JJRcop** [comment]
A symptom that heals cloneloss but only if you're actively bleeding, if your blood_volume drops below BLOOD_LEVEL_OKAY it starts healing brain damage too.

**Comment by PKPenguin321** [comment]
Don't remove weight gain, obesity viruses are funny

**Comment by ChangelingRain** [comment]
We can't not remove weight gain if we want to use hunger as balance for any virus effects.

**Comment by Bawhoppen** [comment]
Make weight gain actually gain weight then, not just replenish hunger.
It's just about how the food they do ingest gets metabolized after all

**Comment by Grazyn** [comment]
Conditional healing of any kind is still pretty OP. Remember how vampires were considered OP as fuck just for having conditional healing (coffins)?

Unlimited healing (as in, healing that isn't limited by your inventory) doesn't really work in our combat system, where most fights are usually determined by attrition. The ability to retreat and heal with no interaction with the surrounding environment should be limited just to some very specific antags.

**Comment by Doppeltongue** [comment]
No, vampires were OP because they could bat form out of any stuns, as well as vent crawl in bat form mode, which is essentially all-access. Our combat system is actually usually determined by stuns. First to stun and keep them stunned wins.

Weight gain isn't as funny now, because fatsprites got removed, so you can't see people being blubbery.

And things we still need healing for: cloneloss, oxyloss, and bloodloss.

**Comment by XDTM** [comment]
Hold on i forgot to change self-respiration, dammit

Also i don't really see many cases where you'd bother with a cloneloss virus, rare as it is.

**Comment by XDTM** [comment]
@Grazyn inventory heals have the benefit of being faster and available in all cases. Conditional virus healing requires you to seek specific conditions, that may be not easily available, especially in a combat situation, and is generally slower.

**Comment by DapperMola** [comment]
This looks like a great way to make virology more interesting! I would love to see this happen!

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 13]
```diff
@@ -305,25 +398,46 @@ Bonus
...
+/datum/symptom/heal/radiation/CanHeal(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	switch(M.radiation)
+		if(0)
+			return FALSE
+		if(1 to RAD_MOB_SAFE)
+			return 0.25
+		if(RAD_MOB_SAFE to 750)
+			return 0.5
+		if(751 to 1250)
```
Comment (JJRcop) [inline code/datums/diseases/advance/symptoms/heal.dm]: Should also be defines

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 14]
```diff
@@ -114,23 +142,39 @@ Regeneration
...
+	stage_speed = -3
+	transmittable = -3
 	level = 6
 
-/datum/symptom/heal/brute/Heal(mob/living/carbon/M, datum/disease/advance/A)
-	var/heal_amt = 2 * power
+/datum/symptom/heal/brute/CanHeal(datum/disease/advance/A)
+	var/mob/living/M = A.affected_mob
+	switch(M.bodytemperature)
+		if(0 to 340)
```
Comment (JJRcop) [inline code/datums/diseases/advance/symptoms/heal.dm]: Should be defines

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 15]
```diff
@@ -114,23 +142,39 @@ Regeneration
...
+			return FALSE
+		if(340 to BODYTEMP_HEAT_DAMAGE_LIMIT)
+			. = 0.3 * power
+		if(BODYTEMP_HEAT_DAMAGE_LIMIT to 400)
+			. = 0.5 * power
+		if(400 to 460)
```
Comment (JJRcop) [inline code/datums/diseases/advance/symptoms/heal.dm]: Should also be defines

**Comment by PKPenguin321** [comment]
>We can't not remove weight gain if we want to use hunger as balance for any virus effects.

We can because:
- It currently only affects 1 (one) symptom in this PR, which is level 8 and arguably not even that good
- It still takes a symptom slot and isn't without downsides (it makes you obese)
- Xhuis has a refactor going up that separates hunger and health, so we could tie the 1 symptom that relies on hunger to health instead provided it goes through

Do not delete weight gain please

**Comment by Grazyn** [comment]
@XDTM drinking water to heal burns. Water is plentiful and readily available. Drinking coffee to heal brute. Yes this is limited, but still extremely easy to find and most people already keep coffee in their backpack in case of lings. 

It still makes healing trivial. It equates healing to hunger, being hurt becomes just a minor inconvenience you can fix anywhere with basically no preparation. Yes, medibots are the same problem, by at least you can emag those.

**Comment by Doppeltongue** [comment]
>no preparation

The virus itself is preparation. A BIG preparation only located in a single part of the station. Water might be a bit difficult, but keeping coffee around to heal brute is no more impressive than keeping a bottle of bicaridine in your backpack.

**Comment by Grazyn** [comment]
How is preparation being passively infected by a self-spreading virus? Because that's the problem with healing viruses, the whole station has them. At least superpowers require someone to hand them out one at a time. And are you seriously saying that bicaridine is as easy to gather as coffee? Come on

**Comment by Doppeltongue** [comment]
THey only self-spread if you sacrifice a symptom to do so. The best healing viruses never did. Bicaridine's slightly harder to get in large quantities than coffee, but remember that any food at all also heals brute. Any poppy-related plant has bicaridine in it, you can trivially ask chemistry for it, and any ambrosia-related plant has healing medicine in it.

Are you seriously equating having some coffee to drink alongside a healing virus when you have time to do so is the same thing as healing while stunned/fighting/crit with no input? Do you complain about people who duck into maint and pull out a patch they got from the public chem fridge? Or those who grow some medicine at the public garden?

**Comment by Jalleo** [comment]
@PKPenguin321 this PR dependson balancing viruses around you having nutrition. Weight gain has to go through nutrition because nutrition is how obesity works.

**Comment by Grazyn** [comment]
@Doppeltongue I never said that the rework is the same as the old healing virus. That said

1. Having a steady supply of drugs on yourself, which means you have to interact regularly with medbay or at least undertake the effort to grow them on your own
2. Heading to the nearest hot drinks vendor/water faucet whenever your health is <100, with no other effort on your part besides a one-time injection (or no effort at all in case of sneezing virus)

If you can't see the difference between these two, there's not much else I can say.

**Comment by XDTM** [comment]
I mean, what's the point of making the virus if it doesn't help you at all? _Someone_ has to put time and effort to make one, it's not like because the receiver only had to get an injection no effort was involved. People can help other people.
Plus, the healing is slower than medbay anyway.

**Comment by Grazyn** [comment]
The problem is that it gives a station-wide bonus to everyone. We already have things that require effort from one/two people and provide a station-wide bonus: they're called station projects, and are late game and very resource-intensive for a reason. Once the initial work is done, everyone gets the benefit with no effort on their part. A healing virus is as powerful as a station project.

This rework is the equivalent of "duck and cover to regenerate health" we see in 2010s vidyas, something I personally dislike as it makes for boring and predictable gameplay.

**Comment by XDTM** [comment]
You're definitely overstating the power of the healing; coffee will take ages to heal you, and tissue hydration reauires you to camp under a shower, instead of popping a pill and healing twice as fast while on the move. These symptoms add strategic choices, they do not replace conventional healing.

**Comment by PKPenguin321** [comment]
@Jalleo Read my first two points again, I'm aware of that but it doesn't matter. Weight gain isn't as much infinite nutrition as it is "you are obese and slow"

**Comment by ChangelingRain** [comment]
weight gain is infinite nutrition that ends up making you slow because you hit the fat benchmark.
the question would be if the metabolic boost was enough to keep you from getting fat with both.

**Comment by XDTM** [comment]
It would not, the numbers on weight symptoms are insanely high; weight loss pretty much increases your hunger rate by ~200 times if you don't count walking, and weight gain just warps you to well fed and then goes up from that. Ypu literally can't outdo it because it has a minimum value.

**Comment by PKPenguin321** [comment]
Yes, so don't remove it please

**Comment by XDTM** [comment]
I don't really like how it works; you can mitigate other symptoms in some ways, by healing damage done, fixing status effects with chems, and so on. But you have no hope of ever going below full with the weight gain symptom, no matter what you do.

**Comment by PKPenguin321** [comment]
As it should be

**Comment by XDTM** [comment]
@KorPhaeron

**Comment by TimeIsEndless** [comment]
Maybe there should be some messages for the healing symptoms. While testing them most of the people who got the viruses didnt even notice and were unable to use the conditional healing.

**Comment by XDTM** [comment]
Good point! I'll add them.

**Comment by XDTM** [comment]
Added messages for viruses while they're healing. I didn't want to push too close to OOCing it, so there still are no messages if you're _not_ under healing conditions, but if you are, you'll know.

**Comment by Arianya** [comment]
One thing that might help make the new diseases more plainly useful is "subtle" messages like some of the old symptoms had so that the infected know with some reasoning what they need to do to activate the symptoms.

Cellular Molding
`You feel your body burn hot.`

Starlight Condensation
`Stars... you want to see them again...`

Toxolysis
`You feel unusually pure.`

 Tissue Hydration
`Your skin feels dry and flaky.`

Plasma Fixation
`The air around you is far too... non-explosive`

Radioactive Resonance
No idea on this one.

Metabolic Boost
`You jitter with nervous energy.`

**Comment by XDTM** [comment]
Added passive messages. Some still require you to be hurt before they show.

**Comment by KorPhaeron** [comment]
Looks more interesting than what we have now, willing to give it a chance. 

Thanks for doing the work on it

**Comment by Hathkar** [comment]
Still needs some improvement, in my opinion. The main problem is that people who do have the healing virus symptoms have no idea how to activate them, and some of the activations are a bit too rare, or detrimental. I'd also go with Arianya's suggestion for subtle messages on what to do for healing, if nothing else is going to change.
Anyway, here's my thoughts on each symptom.

Tissue Hydration: Why bring a water bottle for the healing when you could just fill it with Bicaridine, or some healing patches?

Cellular Molding: Might be okay if fever triggers it, but otherwise, you won't be getting any healing out of it. And if you're caught in an actual fire, the other symptoms won't patch you up enough to live through it.

Toxolysis: I can only see this being used for a virus made to kill people, since it prevents healing chems and slows down the cure from metabolizing. There are very very few cases where I would constantly want ALL Chems to be purged from my system. Sleepers and cryo basically don't do anything when this virus is up.

Starlight Condensation: Practically worthless, if you can trudge over to a window to look into space, you can probably also easily make it to medbay to get charcoal.

Regenerative Coma: A death sentence in a fight. If it start working AFTER you went into crit, it might be worth using.

Plasma Fixation: Great if you're a plasmaman, otherwise if you're standing in a cloud of plasma chances are you'll be burning soon, and are probably better off using internals.

Radioactive Resonance: Again, if you're irradiated, you have larger problems to worry about than healing brute and burn.

Metabolic Boost: Could be decent, but TENFOLD increased hunger? That's honestly ridiculous. Double or triple at most.

In summary, most of these symptoms require are either barely beneficial at best, or actively screw you over at worst.

Why couldn't you just re-work the old symptoms to make you hungry in exchange for healing? Or just cut the healing rate down by half? Or make the healing stop when you take damage for X seconds? Or all three? As of right now, there's very very little reason to play virology except to make meme viruses that grow beards, or death plagues that wipe out the station.

**Comment by ChangelingRain** [comment]
This looks good.

**Comment by XDTM** [comment]
> I'd also go with Arianya's suggestion for subtle messages on what to do for healing, if nothing else is going to change.
Already done.

> Tissue Hydration: Why bring a water bottle for the healing when you could just fill it with Bicaridine, or some healing patches?

You won't always be able to access a full chem dispenser or a nanomed. Always having the option to head to a sink or shower to heal makes you more versatile.

> Cellular Molding: Might be okay if fever triggers it, but otherwise, you won't be getting any healing out of it. And if you're caught in an actual fire, the other symptoms won't patch you up enough to live through it.

You can trigger it with coffee and similar drinks, i think. The fact that fire hurts is balanced by the fact that it can heal up to 6 brute per tick (12 if upgraded), so setting yourself on fire and dousing when healthy IS an option.

> Toxolysis: I can only see this being used for a virus made to kill people, since it prevents healing chems and slows down the cure from metabolizing. There are very very few cases where I would constantly want ALL Chems to be purged from my system. Sleepers and cryo basically don't do anything when this virus is up.

Sleepers have infinite chems, and cryo multiplies input chems. This symptom isn't instant, it's comparable to having charcoal or calomel. And those ARE used for nonantag purposes; aside from poison, think of removing the chems from a sleepypen or a cryosting.

> Starlight Condensation: Practically worthless, if you can trudge over to a window to look into space, you can probably also easily make it to medbay to get charcoal.

Unless medbay is unavailable. If viruses are more efficient than being in the department dedicated to healing, they're overpowered.

> Regenerative Coma: A death sentence in a fight. If it start working AFTER you went into crit, it might be worth using.

Fair balancing, it heals very quickly. It can help a lot against environmental hazards, like fire, space, and door shocks, making you last much longer if not outright saving you. Also the fake death status can fool some attackers.

> Plasma Fixation: Great if you're a plasmaman, otherwise if you're standing in a cloud of plasma chances are you'll be burning soon, and are probably better off using internals.

Plasmamen are immune to viruses anyway. You don't need to breathe plasma to activate this, just be within it, and if the station is plasmafiring you very likely want this, as it will heavily mitigate fire damage.
Drinking some is an extra bonus.

> Radioactive Resonance: Again, if you're irradiated, you have larger problems to worry about than healing brute and burn.

Do you? A mild irradiation is harmless and still works

> Metabolic Boost: Could be decent, but TENFOLD increased hunger? That's honestly ridiculous. Double or triple at most.

Hunger is a joke anyway

> In summary, most of these symptoms require are either barely beneficial at best, or actively screw you over at worst.

They give you more options if you're hurt, instead of making damage 100% trivial.

> Why couldn't you just re-work the old symptoms to make you hungry in exchange for healing? Or just cut the healing rate down by half? Or make the healing stop when you take damage for X seconds? Or all three? As of right now, there's very very little reason to play virology except to make meme viruses that grow beards, or death plagues that wipe out the station.
To not make damage 100% trivial. Again, hunger is a joke and a mild slowdown removed by drinking sugar is not a good balancing point.

**Comment by Hathkar** [comment]
Have you played a round or two with these new virus symptoms? They basically might as well not even exist. This just confirms my belief that coders don't play.

**Comment by KorPhaeron** [comment]
XDTM has 3066 connections in two years

**Comment by Hathkar** [comment]
I don't know why you have such a hateboner for virology, but it's clear to me it's only good for antags now.

**Comment by JJRcop** [comment]
🎉

**Comment by Steelpoint** [comment]
How Virology died, not in a bang, but a mere whimper.
